openwisp_controller/config/static/config/js/widget.js

```diff
@@ -219,6 +241,17 @@
         if (field.attr("data-options") !== undefined) {
             $.extend(options, JSON.parse(field.attr("data-options")));
         }
+        if (getNestedValue(startval, 'interfaces', 0, 'addresses', 0, 'proto') === 'static') {
```

Comment:
we could simply reorder the schema in the python code don't you think? Anyway this won't work if the variable is an IPv6.
And it won't work on any other case but only on this one.

Have you tested the idea I shared previously? That is, we could do something like:

- wait for page load to complete
- reset the initial value again, ensuring the code which removes the maxlength attribute is triggered

Such a workaround would work in different cases and could prove useful until we can rewrite the config editor from scratch, which is something becoming more and more pressuring.